api/advanced-metrics.yaml

```diff
@@ -0,0 +1,70 @@
+paths:
+  /advanced/config:
+    post:
+      tags: ['Advanced Metrics']
+      operationId: set_advanced_metrics
+      description: |-
+        An API that extends the base Traffic Generator configuration with advanced metric configuration.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../advanced/config.yaml#/components/schemas/AdvancedMetrics.Config'
+      responses:
+        '200':
+          $ref: '../result/request.yaml#/components/responses/Success'
+        '409':
+          $ref: '../result/request.yaml#/components/responses/Conflict'
+        '590':
+          $ref: '../result/request.yaml#/components/responses/Fail'
+
+  /advanced/metrics:
+    post:
+      tags: ['Advanced Metrics']
+      operationId: get_advanced_metrics
+      description: |-
+        An API that retrieves advanced metrics.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../advanced/metrics.yaml#/components/schemas/AdvancedMetrics.Request'
+      responses:
+        '200':
+          description: |-
+            Response from traffic generator for chosen metrics
+          content:
+            application/json:
+              schema:
+                $ref: '../advanced/metrics.yaml#/components/schemas/AdvancedMetrics.Response'
+
+
+  /event/subscribe:
+    post:
+      tags: ['Advanced Metrics']
```

Comment:
see my comments earlier about common patterns related to events and generally events != metrics...something to think about.